src/api/test_endpoint.py

```
@@ -25,37 +34,44 @@ def test_health_check_deployed():
             "language": "en",
-            "model": "google_genai:gemini-2.5-flash"
-        }
+            "model": "LongCat-Flash-Chat",
+        },
     }
-    response = requests.post(f"{BASE_URL}/supervisor", json={"users_question": "Where can I buy spirit?", "config": config})
-    print("################# response: ", response)
-    print("################# response.json(): ", response.json()["response"])
+
+    try:
+        response = requests.post(url, json=payload, stream=True)
```

Ruff (0.14.10)

Probable use of `requests` call without timeout

(S113)